lnwallet/commitment.go

```diff
@@ -465,18 +570,53 @@ func CreateCommitTx(chanType channeldb.ChannelType,
 	commitTx := wire.NewMsgTx(2)
 	commitTx.AddTxIn(&fundingOutput)
 
+	var (
+		localOutput  = false
+		remoteOutput = false
+	)
+
 	// Avoid creating dust outputs within the commitment transaction.
 	if amountToLocal >= localChanCfg.DustLimit {
```

Comment:
could assign this condition to `localOutput` and make it `if localOutput`

Reply:
I like it :)